### PR TITLE
doc: add ppp 7

### PR DESCRIPTION
**Pull request by bonomat**
I think we should publish one for this last week :)

### CODE REVIEW

File: docs/ppp/iteration-7-30112022.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Plans 📆
+
+We are pleased that the judges found our project worthy and are pumped to get one more week of hacking in before the final pitch day!
```
Comment (da-kami) [inline docs/ppp/iteration-7-30112022.md:3]:
```suggestion
We are excited that the judges found our project worthy and are pumped to get one more week of hacking in before the final pitch day!
```

File: docs/ppp/iteration-7-30112022.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+For this, we decided to mostly work on UX improvements and get our App into Apple's Testflight and an app store for Android.
+
+A few things on our todo list for this week are:
+
+- Persist lightning payment and bitcoin transaction history
+- Small UI changes
```
Comment (da-kami) [inline docs/ppp/iteration-7-30112022.md:9]:
```suggestion
- UI improvements & polishing
```

File: docs/ppp/iteration-7-30112022.md (hunk 1 above)
Comment (luckysori) [inline docs/ppp/iteration-7-30112022.md:3]: Just pleased?!?

```suggestion
We are thrilled that the judges have deemed our project worthy of the final round and are pumped to get one more week of hacking in before the final pitch day!
```

File: docs/ppp/iteration-7-30112022.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- Simplify on-boarding by funding the app through our own faucet
+- Show transaction details for Lightning as well as for Bitcoin
+- Release version 0.3.0 for Testflight and Android
+
+The main goal of this iteration is to get a usable version out there to on-board early users and collect feedback.
+If you are keen, feel free to reach out.
```
Comment (da-kami) [inline docs/ppp/iteration-7-30112022.md:15]:
```suggestion
If you are keen, please reach out!
```

File: docs/ppp/iteration-7-30112022.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+### Plans 📆
+
+We are pleased that the judges found our project worthy and are pumped to get one more week of hacking in before the final pitch day!
+For this, we decided to mostly work on UX improvements and get our App into Apple's Testflight and an app store for Android.
+
+A few things on our todo list for this week are:
+
+- Persist lightning payment and bitcoin transaction history
```
Comment (luckysori) [inline docs/ppp/iteration-7-30112022.md:8]:
```suggestion
- Persist Lightning payment and Bitcoin transaction history
```

File: docs/ppp/iteration-7-30112022.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+The main goal of this iteration is to get a usable version out there to on-board early users and collect feedback.
+If you are keen, feel free to reach out.
+
+### Progress ✅
+
+Last week was probably one of our most productive ones. Kudos to the whole team.
+
+- Published a blog post about our [vision](https://makers.bolt.fun/story/10101-pitch--441)
+- Created a Pitch video [https://rumble.com/v1wkrpc-10101-legends-of-lightning-pitch.html](https://rumble.com/v1wkrpc-10101-legends-of-lightning-pitch.html)
+- Had a public demo with the community. Recording can be found [here](https://rumble.com/v1wr04i-10101-demo.html)
```
Comment (da-kami) [inline docs/ppp/iteration-7-30112022.md:23]:
```suggestion
- Had a public demo with the community. Recording can be found [here](https://rumble.com/v1wr04i-10101-demo.html), demo summary post can be found [here](https://makers.bolt.fun/story/10101-its-demo-time--470)
```

**Review by holzeis — APPROVED** [review]
Thanks for writing this up. I like the cit. 🙂

File: docs/ppp/iteration-7-30112022.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+- Fix heaps of bugs
```
Comment (luckysori) [inline docs/ppp/iteration-7-30112022.md:24]: 🔧 To keep the tense consistent.

```suggestion
- Fixed heaps of bugs
```

**Review by da-kami — APPROVED** [review]
thanks for writing this up!

File: docs/ppp/iteration-7-30112022.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+- Released version 0.2.1 as an APK downloadable from our [GitHub repository](https://github.com/itchysats/10101/releases)
+- Wired up CFD settlement end-to-end
+
+### Problems ✋
+
+> "Problems are like washing machines. They twist us, spin us and knock us around. But in the end we come out cleaner, brighter and better than before.", Unknown
```
Comment (luckysori) [inline docs/ppp/iteration-7-30112022.md:30]: Hahaha, so good.

**Review by luckysori — APPROVED** [review]
Nice!

**Comment by bonomat** [comment]
Going ahead and publish it.

bors r+

**Pull request merged**